### DISCUSSION TITLE
Add Edit and Delete actions to a View Modal

### DISCUSSION THREAD

**Discussion by duckhunter392** [filamentphp/filament#5983] [Help]
Hello,

I'm looking for some help. I have a Relation Manager that adds a related table to a view page. On the related table, I added a view action to open a modal with the view of record. I want to add an edit and a delete action to this modal.

I wrote the below code, which doesn't work because it's using a table action in a modal. Is there any easy way to add an Edit and a Delete action to a view modal like this?

```php
->actions([
   Tables\Actions\ViewAction::make()
        ->modalActions([            
            Tables\Actions\EditAction::make(),
            Tables\Actions\DeleteAction::make(),
        ]),
])
```

**Comment by danharrin** [comment]
There is not an easy way, they must be in the `$table ->actions()`
